packages/sales-api-service/src/services/recurring-payments.service.js

```diff
@@ -148,3 +148,8 @@ export const cancelRecurringPayment = async id => {
     console.log('No matches found for cancellation')
   }
 }
+
+const determineRecurringPaymentName = (transactionRecord, contact) => {
+  const dueYear = transactionRecord.payment.recurring.nextDueDate.split('-')[0]
```

Comment:
This is really nit-picky, but IMHO it'd be slightly neater to do `const [dueYear] = transactionRecord.payment.recurring.nextDueDate.split('-')`